spec/serializers/api/v2/serializable_provider_spec.rb

```diff
@@ -1,7 +1,9 @@
 require 'rails_helper'
 
 describe API::V2::SerializableProvider do
-  let(:provider) { create :provider, accrediting_provider: 'Y' }
+  let(:accrediting_provider) { create(:provider) }
+  let(:course) { create(:course, accrediting_provider: accrediting_provider) }
+  let(:provider) { create :provider, accrediting_provider: 'Y', courses: [course] }
```

Comment:
Glad to see this coverage extended.

The data is inconsistent, only non-accredited providers would have courses with an accredited body.

how about:

```ruby
let(:provider) { create :provider, courses: [course] }
let(:accrediting_provider) { create(:provider, accrediting_provider: 'Y') }
let(:course) { create(:course, accrediting_provider: accrediting_provider) }
```